xml/common_intro_feedback.xml

```diff
@@ -15,7 +15,7 @@
  <title>Giving Feedback</title>
 
  <para>
-  We welcome feedback on and contributions to this documentation.
+  We welcome feedback for and contributions to this documentation.
```

Comment:
TBH, I preferred "on".

Reply:
Uups, that was by accident and should be committed. Fixed it with your suggestion in commit 5cbc528